docs/UserGuide.md

```diff
@@ -435,7 +457,8 @@ Examples:
 
 #### 4.2.6 Clearing all client data : `clear`
 
-Clears all data from the DonnaFin.
+Clears all data from the DonnaFin. This is a destructive command that cannot be recovered from, please take care when
```

Comment:
can use the caution block for "This is a destructive command that cannot be recovered from, please take care when..."

Reply:
Done!